clients/ts/signalr/src/NodeHttpClient.ts

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 import * as http from "http";
+import * as https from "http";
```

Comment:
I think this should be:
```suggestion
import * as https from "https";
```

Reply:
Whoops, copy-pasta error, thanks